.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -64,10 +63,17 @@ body:
     required: true
 - type: textarea
   attributes:
-    label: Anything else?
+    label: Trace Logs?
     description: |
       Trace Logs (https://wiki.servarr.com/radarr/troubleshooting#logging-and-log-files) 
-      Links? References? Anything that will give us more context about the issue you are encountering!
+      ***Generally speaking, all bug reports must have trace logs provided.***
```

Comment:
We don't need a whole new input, just leave as is and keep the title change.

Reply:
done